Renoon-same/frontend
# Authentication Page: Sign In, Sign Up, Forgot password

# Description
- Implement form validation for sign in , sign up, reset password
- Download react-hot-toast package
- Implement hot toast for message notification

## Related PRs

List any other PRs that this PR is related to (in case of FE + BE changes).

## Type of change

### Please delete options that are not relevant

## Screenshots/Recordings
![Screenshot 2024-07-07 at 8 28 06 PM](https://github.com/Renoon-same/frontend/assets/122650930/735b200a-c538-4c15-98af-0c637b4c31c3)
![image](https://github.com/Renoon-same/frontend/assets/122650930/bd233b73-8b66-4074-b1ff-63acb45ef8c1)
![image](https://github.com/Renoon-same/frontend/assets/122650930/5d43cdc6-55b7-4129-a142-67ea5fb55ad7)


## Testing instructions

Instructions on how to test the changes made in the PR, helping reviewers validate the code.

1. Run `git checkout feature/authentication`.
2. Run `npm install` and `npm run dev`.
3. Navigate to route `http://localhost:5173/account/registration`
4. Verify that form validation works as expected.
